### PR TITLE
Adding implementation for placeValidFromOrders to ExchangeApi

### PR DESCRIPTION
**Pull request by alfetopito**
Adding implementation for `placeValidFromOrders` to ExchangeApi

Requirement for #402

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr420--dexreact.review.gnosisdev.com

File: src/api/exchange/ExchangeApi.ts [hunk 1]
```diff
@@ -169,6 +181,50 @@ export class ExchangeApiImpl extends DepositApiImpl implements ExchangeApi {
...
+    assert(
+      [sellTokens, validFroms, validUntils, buyAmounts, sellAmounts].every(el => el.length === length),
+      'Parameters length do not match',
+    )
+    assert(length > 0, 'At least one order required')
+
+    const contract = await this._getContract(networkId)
+
+    const buyAmountsStr = buyAmounts.map(amountBn => amountBn.toString())
+    const sellAmountsStr = sellAmounts.map(amountBn => amountBn.toString())
```
Comment (Velenir) [inline src/api/exchange/ExchangeApi.ts]: Can do `sellAmounts.map(String)`, not that it matters

**Comment by alfetopito** [comment]
> I like it!
> 
> One question. Although the interface in solidity has many arrays, don't you think that if we create our API for it it makes more sense to define it as an array of orders with buyToken, sellToken, etc.
> It feels more natural imo.

We definitely can do that and it's somehow what has been done on #425 but in a hook.

The reason I didn't do that in the API, is mainly to keep the same standard, as most (all?) other API methods follow very closely the contract interface.

**Comment by anxolin** [comment]
Okay, fair enough. It's true that if we get too far from the implementation signature, we end up abstracting too much.
If we just work with the hook in the UI and the hook is simpler to use, I think it's good enough. Thx!

**Pull request merged**